homeassistant/components/websocket_api/http.py

```diff
@@ -68,15 +66,16 @@ class WebSocketHandler:
 
     def __init__(self, hass: HomeAssistant, request: web.Request) -> None:
         """Initialize an active connection."""
-        self.hass = hass
-        self.request = request
-        self.wsock = web.WebSocketResponse(heartbeat=55)
+        self._hass = hass
+        self._request: web.Request = request
+        self._wsock = web.WebSocketResponse(heartbeat=55)
         self._handle_task: asyncio.Task | None = None
         self._writer_task: asyncio.Task | None = None
         self._closing: bool = False
+        self._authenticated: bool = False
```

Comment:
This is just added for the `repr` for debugging, right?

Reply:
Yes, I wanted to know if the leak happens before or after the auth phase